Project.toml

```diff
@@ -19,13 +19,15 @@ julia = "1"
 
 [extensions]
 MeasurementsJunoExt = "Juno"
+MeasurementsMakieExt = "Makie"
 MeasurementsRecipesBaseExt = "RecipesBase"
 MeasurementsSpecialFunctionsExt = "SpecialFunctions"
 MeasurementsUnitfulExt = "Unitful"
 
 [extras]
 Aqua = "4c88cf16-eb10-579e-8560-4a9242c79595"
 Juno = "e5e0dc1b-0480-54bc-9374-aad01c23163d"
+Makie = "ee78f7c6-11fb-53f2-987a-cfe4a2b5a57a"
```

Comment:
This isn't used, is it?

Reply:
not at the moment, ideally, i would like to add some tests, but i'm not familiar enough with the makie testing infraestructure